src/ExpressiveInstaller/OptionalPackages.php

```diff
@@ -350,12 +336,8 @@ public function promptForOptionalPackages()
 
     /**
      * Prompt for a single optional installation package.
-     *
-     * @param string $questionName Name of question
-     * @param array $question Question details from configuration
```

Comment:
It would be nice to have doc comments when it contains some important information. Not sure if in this case it contains something important, usually I left them when there is a param description.